server/seed/seed.js

```diff
@@ -0,0 +1,133 @@
+const db = require('../dbconnection/schema.js');
```

Comment:
when you run your seed script, it will add onto the current database and increase the amount of rows. Ask around, preferably a tech mentor or tech staff, if you need to empty your database  so that you will have a constant amount of rows in your db everytime you run script

Reply:
okay will do